clients/js/src/consts/networks.ts

```diff
@@ -171,47 +171,72 @@ const MAINNET = {
     key: getEnvVar("ETH_KEY"),
     chain_id: 30,
   },
-  sepolia: {
```

Comment:
Sorry this looks so ugly. All I really did was pull Scroll and Mantle up to put them in the right place based on chain ID.